### PR TITLE
reference: rename TiDB Controller to TiDB Control

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### What is changed, added or deleted? (Required)

Rename the title, make it the same as others.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: reference/tools/tidb-control.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ---
-title: TiDB Controller 使用说明
+title: TiDB Control 使用说明
 category: reference
-aliases: ['/docs-cn/tools/tidb-controller/']
+aliases: ['/docs-cn/tools/tidb-control/']
```
Comment (TomShawn) [inline reference/tools/tidb-control.md:4]: Alias 改动的话，点开这个链接会 404。
```suggestion
aliases: ['/docs-cn/tools/tidb-controller/']
```

Comment (jackysp) [reply reference/tools/tidb-control.md:4]: 难道 alias 是一次性的？不太科学吧。

Comment (TomShawn) [reply reference/tools/tidb-control.md:4]: alias 是这个文档以前改动过文件名。用户如果访问以前的旧链接，会自动跳转到现在的链接。这里的 alias 是旧链接中包含的文件名，如果改了的话，旧链接就会变成死链。

Comment (jackysp) [reply reference/tools/tidb-control.md:4]: got

**Pull request merged**